recipe/meta.yaml

```diff
@@ -12,6 +12,7 @@ source:
 build:
   noarch: python
   number: 0
+  skip: True # [s390x]
   script: {{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
```

Comment:
Currently the package is failing for the `ppc64le` architecture
We might need to add a `skip` option for the `ppc64le` and add a comment explaining that we are not supporting that architecture at this point.

Reply:
Thank you @varlackc for the feedback. A bit strange that the `ppc64le` has failed, as it was not failing before. I will double check with @katietz before.

Reply:
The problem is caused by Prefect. Will close and reopen PR to restart the builds.